examples/README.md

```diff
@@ -4,6 +4,12 @@
   - Learn how to write an HTTP service using annotations.
   - See [Annotated services](https://line.github.io/armeria/server-annotated-service.html).
 
+- `context-propagation`
+  - Learn how to propagate Armeria's `RequestContext` for use in scenarios like tracing.
+  - [`dagger`](https://dagger.dev) provides an example using the Dagger asynchronous framework for
```

Comment:
Better hyperlinking for `the Dagger asynchronous framework`?

Reply:
Yup